plugins/woocommerce-blocks/assets/js/data/cart/default-state.ts

```diff
@@ -94,6 +95,7 @@ export const defaultCartState: CartState = {
 		paymentMethods: EMPTY_PAYMENT_METHODS,
 		paymentRequirements: EMPTY_PAYMENT_REQUIREMENTS,
 		extensions: EMPTY_EXTENSIONS,
+		...persistenceLayer.get(),
```

Comment:
Uses the data from local storage to populate the default state of the cart before resolution.